script/package-dist.js

```diff
@@ -2,78 +2,39 @@
 'use strict';
 
 const path = require('path');
-const promise = require('es6-promise');
-const promisify = require('es6-promisify');
-const promisifyAll = require('es6-promisify-all');
-const fs = promisifyAll(require('fs-extra'));
 const archiver = require('archiver');
+const fs = require('fs');
 
-const pkg = require('../package.json');
-const version = pkg.version;
-const name = pkg.name;
+const { name, version } = require('../package.json');
+const src = path.resolve(__dirname, '../dist');
+const dst = path.join(path.resolve(__dirname, '..'), `${name}-${version}`);
 
-const from = path.resolve(__dirname, '../dist');
-const tmp = path.resolve(__dirname, '../_dist');
-const to = path.join(path.resolve(__dirname, '../dist'), name, version);
-const distName = path.join(path.resolve(__dirname, '..'), name + '-' + version);
-
-const createArchive = (archive, src, dst) =>
-{
-	return new Promise((resolve, reject) =>
-	{
-		const output = fs.createWriteStream(dst);
-
-		output.on('close', () =>
-		{
-			resolve();
-		});
-
-		archive.on('error', (err) =>
-		{
-			reject(err);
-		});
+const createArchive = (archive, src, compressedFile) =>
+	new Promise((resolve, reject) => {
+		const output = fs.createWriteStream(compressedFile);
+		output.on('close', () => resolve());
 
+		archive.on('error', (err) => reject(err));
 		archive.pipe(output);
-
-		archive
-			.bulk([
-				{ expand: true, cwd: src, src: ['**/*.*'] }
-			])
-			.finalize();
-	})
-};
-
-const createDistTar = () =>
-{
-	const archive = archiver('tar', {
-		gzip: true,
-		gzipOptions: {
-			level: 1
-		}
+		archive.directory(src, '');
+		archive.finalize();
 	});
 
-	return createArchive(archive, to, distName + '.tar.gz');
-};
-
-const createDistZip = () =>
-{
-	const archive = archiver('zip', {});
-	return createArchive(archive, to, distName + '.zip');
-};
+const createDistTar = () => createArchive(
+	archiver('tar', { gzip: true, gzipOptions: { level: 9 } }),
+	src,
+	`${dst}.tar.gz`
+);
 
-const createEs6Zip = () =>
```

Comment:
Why is es6 removed? Or is the normal output now es6 by default?

Reply:
Not really know why it was there I just followed the output of the current builds which contain a folder with ES6. We could however package the ES6 version in a seperate package.

Reply:
It was there to be downloadable from an external location.
All other exports (amd, umd, commonjs, browser) are exported to a single file. However, es6 consists of multiple files, so when creating a single zip you can download that version.

Reply:
Updated as discussed!